CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.41 (05/08/2020)
+
+### Changes
+
+- feat(profiling): reduce the default amount of sampling for memory/lock to 2% (#1586)
```

Comment:
```suggestion
- feat: add asgi integration (#1567)
- feat(profiling): reduce the default amount of sampling for memory/lock to 2% (#1586)
```

Reply:
yep, it was missing from the milestone. Thanks!